test/Quest.spec.ts

```diff
@@ -427,4 +432,59 @@ describe('Quest', async () => {
       )
     })
   })
+
+  describe('erc20Stream Quest', async () => {
```

Comment:
Might be good to test the Sablier address upgrades just to make sure the permissions and flow around that upgrade perform as expected.